tests/test_web.py

```diff
@@ -297,34 +297,34 @@ def test_web_submit_no_file(self):
             # Will only pass when there is network connectivity. TODO: Mock external API response
             # self.assertTrue("Did not send to autograder" not in self.driver.page_source)
 
-        def test_staff_submit(self):
-            self._seed_course()
+        # def test_staff_submit(self):
+        #     self._seed_course()
 
-            self._login_as(email=self.staff1.email)
-            self.page_load("{}/admin/course/{}/{}/{}/submit".format(
-                self.get_server_url(),
-                self.course.id,
-                self.user1.email,
-                self.assignment.id,
-            ))
+        #     self._login_as(email=self.staff1.email)
+        #     self.page_load("{}/admin/course/{}/{}/{}/submit".format(
+        #         self.get_server_url(),
+        #         self.course.id,
```

Comment:
No particular reason that this was commented out. It could be kept - but all of the other file upload tests are commented out too.